addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -124,8 +125,8 @@ const MoveArticleToDialog = Dialog.extend({
                             children: data.results.map(record => {
                                 return {
                                     id: record.id,
-                                    icon: record.icon,
-                                    text: record.name,
+                                    text: record.display_name,
+                                    subject: record.root_article_id,
```

Comment:
Isn't root article id going to be a name_get result (id, name) ? Seems we could properly handle False or tuple result and assign it cleanly to variables.